torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -383,16 +306,24 @@ int THPVariable_setitem(PyObject* self, PyObject* index, PyObject* py_value) {
     // real 0-sized shapes.
     return 0;
   } else if (index == Py_Ellipsis) {
-    copy_to(self_, value);
+    at::indexing::copy_to(self_, value);
     return 0;
   } else if (index == Py_None || index == Py_True) { // NOLINT(cppcoreguidelines-pro-type-cstyle-cast)
-    copy_to(self_.unsqueeze(0), value);
+    at::indexing::copy_to(at::indexing::handleNoneSingleDim(self_), value);
```

Comment:
I really feel like we are getting zero utility from the `handleNoneSingleDim` function here. The inlined version `self_.unsqueeze(0)` even is shorter...